kube/secret.go

```diff
@@ -50,9 +60,27 @@ func MakeSecrets(secrets model.CVMap, defaults map[string]string) ([]*apiv1.Secr
 	total := 0 // Accumulated size of the values stored in 'currentSecret'
 
 	for key, value := range secrets {
-		ok, strValue := configValue(value, defaults)
-		if !ok {
-			return nil, nil, fmt.Errorf("Secret '%s' has no value", key)
+		var strValue string
+		if createHelmChart {
+			switch {
+			case value.Generator == nil || value.Generator.Type != model.GeneratorTypePassword:
+				errString := fmt.Sprintf("A valid .Values.env.%s is required", value.Name)
+				strValue = fmt.Sprintf(`{{ required "%s" .Values.env.%s | b64enc | quote }}`, errString, value.Name)
+			case value.Generator.Type == model.GeneratorTypePassword:
+				strValue = "{{ randAlphaNum 32 | b64enc | quote }}"
+			default:
```

Comment:
It can't; this is leftover code from when the certs where not treated as config vars, but included literally. I'll remove it.